index.html

```diff
@@ -1,94 +1,132 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-        <!-- Global site tag (gtag.js) - Google Analytics -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=G-7YM82MKHNB"></script>
-        <script>
-          window.dataLayer = window.dataLayer || [];
-          function gtag(){dataLayer.push(arguments);}
-          gtag('js', new Date());
-    
-          gtag('config', 'G-7YM82MKHNB');
-        </script>
+    <meta charset="utf-8" />
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-7YM82MKHNB"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
 
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300&family=Roboto:wght@100;500&display=swap" rel="stylesheet">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      gtag("config", "G-7YM82MKHNB");
+    </script>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300&family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Walter Velasquez</title>
     <link rel="stylesheet" href="css/style.css" />
-    <script src="https://kit.fontawesome.com/62ea397d3a.js" crossorigin="anonymous"></script>
+    <script
+      src="https://kit.fontawesome.com/62ea397d3a.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
- 
+
   <body>
     <header>
+      <div class="logo">Sir V</div>
       <nav class="nav">
+        
         <div class="nav_mobile">
-          <div class="logo">Sir V</div>
-          <div class="nav_mobile-btn 
-          fas fa-bars"></div>
+          <div id="hamburguer" class="nav_mobile-btn fas fa-bars"></div>
         </div>
       </nav>
     </header>
-
-    <section class="headline">
+    <a id="headline_seccion"></a>
+    <section class="headline" id="headline">
       <div class="presentation">
         <h1>Walter Velasquez</h1>
         <h2>Software Developer</h2>
       </div>
```

Comment:
- [ ] I would strongly recommend fixing the width of the heading section to improve your design and make it more professional.
When I open your portfolio I have to scroll the page horizontally to find the hamburger button which is a bad user experience

Your design before scrolling the page horizontally:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/152059501-e6d6e68b-2c9e-40a6-ab28-4f540e3a2468.png">

Your design after scrolling the page horizontally:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/77190203/152058884-517d9ccc-0a8f-46fd-89e5-14f5011719ca.png">

Reply:
- [ ] [REQUIRED] As stated by the previous reviewer, kindly fix the width of the heading as I also had to scroll horizontally to locate the hamburger icon. Please fix this to make your project look more professional.

Here is an image of what it looks like after scrolling horizontally:
![Screenshot from 2022-02-02 16-19-27](https://user-images.githubusercontent.com/58692590/152183381-6e3fdda3-d067-444f-bc5c-6d8be3ea065f.png)